src/main/resources/assets/nochatreports/lang/en_us.json

```diff
@@ -10,5 +10,39 @@
+	"option.NoChatReports.enableDebugLog": "Enable debug log",
+	"option.NoChatReports.enableDebugLog.tooltip": "True if mod should log some additional information about its activity and message signatures.\nGenerally only useful for debugging.",
+	"option.NoChatReports.convertToGameMessage": "Convert to game message",
+	"option.NoChatReports.convertToGameMessage.tooltip": "True if server should convert all player messages to system messages.",
+	"option.NoChatReports.showServerSafety":"Show server safety",
+	"option.NoChatReports.showServerSafety.tooltip":"True if current server safety level should be indicated on client by an icon in the bottom-right corner of chat screen.",
+	"option.NoChatReports.hideRedChatIndicators":"Hide red chat indicators",
+	"option.NoChatReports.hideRedChatIndicators.tooltip":"True if vanilla's unsigned messages trust indicator should be removed.",
+	"option.NoChatReports.hideYellowChatIndicators":"Hide yellow chat indicators",
+	"option.NoChatReports.hideYellowChatIndicators.tooltip":"True if vanilla's modified/hidden messages trust indicator should be removed.",
+	"option.NoChatReports.hideGrayChatIndicators":"Hide gray chat indicators",
+	"option.NoChatReports.hideGrayChatIndicators.tooltip":"True if vanilla's system messages trust indicator should be removed.",
+	"option.NoChatReports.hideWarningToast":"Hide warning toast",
+	"option.NoChatReports.hideWarningToast.tooltip":"True if \"Chat messages can't be verified\" toast should be removed.",
+	"option.NoChatReports.alwaysHideReportButton":"Always hide report button",
+	"option.NoChatReports.alwaysHideReportButton.tooltip":"True if reporting buttons should always be hidden from social interactions screen.",
+	"option.NoChatReports.disableTelemetry":"Disable telemetry",
+	"option.NoChatReports.disableTelemetry.tooltip":"True if sending telemetry to Mojang should be disabled.",
+	"option.NoChatReports.showReloadButton":"Show reload button",
+	"option.NoChatReports.showReloadButton.tooltip":"True if the configuration reload button should be displayed in multiplayer screen.",
+	"option.NoChatReports.whitelistAllServers":"Whitelist all servers",
+	"option.NoChatReports.whitelistAllServers.tooltip":"True if the full-screen warning should be skipped for servers that require chat signing, letting users simply join the server.\nBecause of that, servers will no longer be added to the whitelist.",
+	"option.NoChatReports.whitelistedServers":"Whitelisted Servers",
+	"option.NoChatReports.whitelistedServers.tooltip":"Current list of whitelisted unsafe servers.\nUsed on client to remember which servers user disabled unsafety warnings for.",
+	"option.NoChatReports.whitelistedServers.addButtonTooltip":"Add server",
+	"option.NoChatReports.whitelistedServers.removeButtonTooltip":"Remove server",
+	"option.NoChatReports.whitelistedServers.instructions":"When adding a server click below Whitelisted Servers to type in the field."
+}
```

Comment:
I would like you to update config option descriptions with respect to the documentation I have written on them (to possible extent): https://github.com/Aizistral-Studios/No-Chat-Reports/wiki/Configuration-Files/

Reply:
Got it done.